index.html

```diff
@@ -21,6 +23,13 @@
         <p>Pat's Salmon Cookies</p>
         <img id="salmon" src="https://github.com/codefellows/seattle-201d8/blob/master/class-09-advanced_css_layout/lab/assets/salmon.png?raw=true">
       </div>
+
+    <section id = "testimonials">
+      <img src="https://github.com/codefellows/seattle-201d8/blob/master/class-09-advanced_css_layout/lab/assets/cutter.jpeg?raw=true">
+
+      <p>Seattle's Original Salmon Cookies</p>
```

Comment:
@AlisonZ no space needed here - all direct children of the container section should touch - no line spacing needed